src/shared/local-addresses.c

```diff
@@ -211,7 +211,7 @@ int local_addresses(
                         break;
 
                 default:
-                        continue;
+                        assert_not_reached();
```

Comment:
No, the message is passed from the kernel. This condition may hit if the kernel sends an address currently we do not support. We always need to handle such cases gracefully.

Reply:
This is already explicitly checked above.

```c
if (!IN_SET(family, AF_INET, AF_INET6))
        continue;
```

Reply:
Ah, indeed. Then. LGTM.